### PR TITLE
admin can check own task and all task, Manager can check own task and…

### PR DESCRIPTION
**Pull request by harmeshverma01**
… that particular task which they assign to user

### CODE REVIEW

File: task/views.py [hunk 1]
```diff
@@ -54,16 +54,19 @@ class CheckTaskView(APIView):
     
     def get(self, request, id=None):
         params = self.request.query_params
-        task = Task.objects.all()
-        manager = params.get('manager', None)
+        assign = Task.objects.all()
+        admin = request.user.id
```
Comment (nodeexcel) [inline task/views.py:58]: remove unnecessary variable ( admin = request.user.id ) 
remove extra if else

File: task/views.py [hunk 2]
```diff
@@ -89,17 +92,20 @@ class ManagerCheckTaskView(APIView):
     
     def get(self, request, id=None):
         params = self.request.query_params
-        task = Task.objects.all()
+        assign = Task.objects.all()
         manager = request.user.id
         if manager:
```
Comment (nodeexcel) [inline task/views.py:97]: same like above, remove 
remove unnecessary variable 
remove extra if else

File: task/views.py (hunk 1 above)
Comment (harmeshverma01) [reply task/views.py:58]: fixed

File: task/views.py (hunk 2 above)
Comment (harmeshverma01) [reply task/views.py:97]: fixed

**Pull request merged**